bot.py

```diff
@@ -171,7 +168,7 @@ async def set_leaderboard_channel(ctx, channel: discord.TextChannel):
     await ctx.respond("Leaderboard channel updated.")
 
 @commands.has_role("Staff")
-@bot.slash_command(description="Staff only command. Sets the leaderboard output channel.")
+@bot.slash_command(description="Staff only command. Sets the soloque leaderboard output channel.")
 async def refresh_soloque_channel(ctx: discord.ApplicationContext):
     global solo_queue_leaderboard
     if solo_queue_leaderboard == None:
```

Comment:
I'm pretty sure these instances now need to be `inhouse.global_objects.solo_queue_leaderboard` to make sure we're referencing the correct thing.